build.sbt

```diff
@@ -1,26 +1,24 @@
-inThisBuild(List(
```

Comment:
why did we move away from `inThisBuild` to prepending `ThisBuild` on every line? Admittedly I am not super familiar, but it seems they're functionally equivalent according to [these sbt docs](https://www.scala-sbt.org/1.x/docs/Scopes.html#Build-level+settings)?

Reply:
That's a good question. I've found that the `inThisBuild` function doesn't work in all cases, and it's more reliable to use the `ThisBuild / ` prefix. It's been a while since I've had problems with `inThisBuild` though, so it might be a little bit of cargo-culting on my part at this point?

(IIRC the problems come up when sbt plugin authors don't use sbt scopes correctly, which is an easy thing to get wrong when you're starting off writing plugins. I'm not really sure why one works and the other doesn't, though.)